vaadinfrontend/frontend/themes/datamanager/components/custom.css

```diff
@@ -2,6 +2,7 @@
 
 /*Make sure to import the css classes for each custom component to avoid cluttering the master-style-sheet */
 @import "card.css";
+@import "page-area.css";
```

Comment:
Hi @wow-such-code 
As the last definition always has precedence, I think it would be good to have the most specific CSS at the bottom and general CSS on top. So reversing the order of the import might matter